### DISCUSSION TITLE
Delta in heat pump heating

### DISCUSSION THREAD

**Discussion by krzbor** [Egyras/HeishaMon#562] [Q&A] [answered]
I have a question regarding the delta in heating. I have a Series K pump. The temperature is set to 22°C, and the delta is 2°C. The heating turns off at 24°C and turns on at 19°C. How is this calculated? Because I get +2°C and -3°C.

**Comment by MiG-41** [comment]
How your pomp is set ? Is it temperature of water outlet , or yuo are using internal thermostat ?

**Reply by krzbor** [reply]
panasonic_heat_pump/main/Main_Outlet_Temp

**Comment by MiG-41** [comment]
So what is your temperature in rooms then , if your outlet temp is max 24 deg ( 3min and off ,so more of the time 23 or less).

As i recall this function correctly..... So when your target temp is 22 , HP starts modulating to keep it. When modulation is not enough , and temperature reach more then 2deg of target ( basicly 2.25deg is enough) then after 3min HP will switch off ( or temp will drop to 2 deg difference or less).  But during stop HP remembers the Inlet temperature. And this Inlet temperature is now reference , and have to drop again 2 deg before this reference  to turn compressor on.

So it could be like this , that your outlet is like 24,25deg , but your inlet has only 21deg , and HP wait to have it 19deg to turn on again...

**Reply by krzbor** [reply]
The house is unoccupied, so I set the minimum possible outlet temperature to 20°C. With a delta of 3°C, the pump turned on at 16°C, but something strange happened.
Despite the positive outside temperature, the pump activated the internal heater. The heater only turned off when the return temperature reached 19.75°C.
The heating was completely uneconomical (COP: 1.9). I considered this a pump error—perhaps it determined that the return temperature was far too low and decided to raise it at any cost,
even though under the current conditions it could have done this economically with a COP of 7 and without the heater.
After this experience, I raised the temperature to 22°C and reduced the delta to 2°C to prevent the situation from happening again.
However, I became curious about the impact of the delta—specifically, what is compared and controlled to turn the compressor on or off

**Reply by MiG-41** [reply]
Maybe turning off the heater in service menu will do this job.

**Reply by geduxas** [reply]
It would be much simpler if you post your graphs, inlet/outlet temperature, target temperature, compressor status, and heater status, do you have some graphs?

**Reply by krzbor** [reply]
Perhaps I was too quick to point out an error. I need to carefully analyze my pump settings related to the possibility of activating the internal heater. According to the manual, there are as many as four control parameters.

**Reply by geduxas** [reply]
I heard some rumors from daikin community, that pump decides in which condition it would be more economically to start compressor or just quickly use heaters.. it's possible could be new features in K/L models.. they're pretty new.. maybe it's first season for them :)

**Reply by krzbor** [reply]
![tem](https://github.com/user-attachments/assets/fd919c69-4437-4cfd-a1a9-da2d76c81aa2)

**Reply by MiG-41** [reply]
No , not this menu... 
Serwice menu to enter while HP turned off by holding 10s LEFT-DOWN-MENU

**Reply by geduxas** [reply]
@MiG-41 it should be done with caution.. it's not recommended to disable heaters.

**Comment by geduxas** [comment]
Delta is only to maintain inlet/outlet proportion.. it will drive circulation pump to maintain that different. And off indicator is inlet temperature.. if it reaches outlet temperature then it indicate pump to switch off. 

Whole logic is very well described in service manual.

**Reply by krzbor** [reply]
You wrote what the condition for turning off the pump is. My question is—what is the condition for turning it back on?

**Reply by geduxas** [reply, marked as answer]
Here is snipet from service manual:

- Compressor is OFF when Water Outlet Temperature – Internal Water Setting Temperature > 2°C for 
continuously 3 minutes. 
-  Compressor is ON after waiting for 3 minutes, if the Water Outlet Temperature – Water Inlet Temperature 
(temperature at thermostat OFF is triggered) <-3°C

**Reply by geduxas** [reply]
And this from H series service manual, it might be that in K it changed.. or left the same logic..

**Reply by krzbor** [reply]
Thank you for the explanations! As you can see, this is exactly how my pump behaves, and I mistakenly connected it to the delta.

**Comment by krzbor** [comment]
I decided to analyze the condition for restarting the pump:

Water Outlet Temperature – Water Inlet Temperature
(temperature at thermostat OFF is triggered) < -3°C

Substituting, we get:
Water Outlet Temperature – (Internal Water Setting Temperature + 2°C - delta) < -3°C

After rearranging the formula, we get:
Water Outlet Temperature – Internal Water Setting Temperature < -1°C - delta

So it seems I wasn’t mistaken after all—delta also has an impact on turning the pump back on.

**Reply by MiG-41** [reply]
Yes , but real delta , with is not always the same ,with is set in HP ( HP not always is stick to preset one, especially during modulation). Also there is preset like Max. Flow instead setting delta...

**Reply by krzbor** [reply]
Yes, I know that the rearranged formula is not precise because the actual delta at the time of shutdown may be different (usually smaller) than the set one.
Nevertheless, delta does affect the pump's activation temperature. The formula after transformation shows that if the delta is, for example, 4°C, then the activation condition will be:
Water Outlet Temperature – Internal Water Setting Temperature < -5°C
With the default internal heater settings (-4°C) and if other conditions are met (time, outside temperature, etc.), the heater might activate.
Therefore, the delta for heater activation should be 2°C lower (-6°C) than the heating delta. This ensures that the heater will only activate when the compressor is unable to handle the heating

**Reply by MiG-41** [reply]
In H version delta for heater is not setable , it is hardcoded for 8...

**Reply by geduxas** [reply]
@krzbor it's totally wrong. Delta is only to maintain flow.

**Reply by geduxas** [reply]
Delta, is difference between supply an inlet which pump trying to maintain. Outlet is target setpoint, and to keep inlet in delta range, pump control it's flow.. and temperature difference with heat capacity is heating energy which your house consumes.. by that compressor is modulated to not generate more than your house consumes... 
But it's not ideal, and compressor maybe produces mire energy.. so inlet starts increase. And pimp shutting down.

**Reply by krzbor** [reply]
Did you analyze my formula thoroughly? It is precisely about this:

Water Inlet Temperature (temperature at thermostat OFF is triggered) = Internal Water Setting Temperature + 2°C - delta

**Reply by geduxas** [reply]
No, it's not like that. Off condition is not related to delta at all.

**Reply by geduxas** [reply]
Sorry, i little bit confused my self, and you, yes and no.. in theory you are right, maybe in most conditions.. it depends in some situations :)

**Reply by krzbor** [reply]
I will describe my calculations in detail.

The condition for switching off is:
Water Outlet Temperature – Internal Water Setting Temperature > 2°C

Additionally:
delta = Water Outlet Temperature - Water Inlet Temperature

Therefore:
Water Inlet Temperature + delta - Internal Water Setting Temperature > 2°C

For the calculations, it’s better to assume equality (switching point):
Water Inlet Temperature + delta - Internal Water Setting Temperature = 2°C

Next, we have:
Water Inlet Temperature = 2°C + Internal Water Setting Temperature - delta
This is the compressor shut-off point, hence:

Water Inlet Temperature = 2°C + Internal Water Setting Temperature - delta = Water Inlet Temperature (temperature at thermostat OFF is triggered)

Now, we use the second formula:
Water Outlet Temperature – Water Inlet Temperature (temperature at thermostat OFF is triggered) < -3°C

Substituting, we get:
Water Outlet Temperature – (2°C + Internal Water Setting Temperature - delta) < -3°C

Which simplifies to:
Water Outlet Temperature - 2°C - Internal Water Setting Temperature + delta < -3°C

Finally, the condition for switching on again:
Water Outlet Temperature - Internal Water Setting Temperature < -1°C - delta

This is not an exact switching point, as the pump usually has a smaller delta when stopping, but in general, this is how it should work, or at least that’s what the Panasonic formulas suggest.
